docs/modules/ROOT/pages/usage.adoc

```diff
@@ -14,6 +14,9 @@ To use the LFV Reqstool Java Maven Plugin, add the following configuration to yo
                     <goals>
                         <goal>assemble-and-attach-zip-artifact</goal>
                     </goals>
+                    <!-- The plugin runs in the 'verify' phase by default. -->
+                    <!-- You can override this by uncommenting and modifying the phase below: -->
+                    <!--<phase>verify</phase>-->
```

Comment:
Not a good comment because the plugin the won't work then. Don't set users up to fail.

Instead explain why the verify is used if anything.